cmd/collector/app/metrics.go

```diff
@@ -28,6 +28,8 @@ const (
 
 	// otherServices is the catch-all label when number of services exceeds maxServiceNames
 	otherServices = "other-services"
+
+	samplerTypeKey = "samplerType"
```

Comment:
let's use `sampler_type` since the metrics library we use will lowercase this when it's reported.